bazel/bazel.go

```diff
@@ -104,6 +105,7 @@ type bazel struct {
 	args          []string
 	ctx           context.Context
 	cancel        context.CancelFunc
+	startupArgs   []string
```

Comment:
Nit: coudl you group args and startupArgs together then put a newline after

```
args        []string
startupArgs []string

ctx    context.context
cancel context.CancelFunc
```

Reply:
Fixed.